docs/guide/overview.asciidoc

```diff
@@ -0,0 +1,86 @@
+`requirements.txt` is::
+
+    # Elasticsearch 7.x
+    eland>=7,<8
+
+Because Eland uses some experimental APIs for Machine Learning it
+is also recommended to install the same major and minor for `elasticsearch-py`
+as your cluster. For example if your cluster is v7.8.1 you would install
+like so::
+
+    $ python -m pip install 'eland>=7,<8' 'elasticsearch>=7.8,<7.9'
+
+[discrete]
+=== Getting Started
+
+Create a `DataFrame` object connected to an Elasticsearch cluster running on `localhost:9200`:
+
+[source,python]
+------------------------------------
+>>> import eland as ed
+>>> df = ed.DataFrame(
+...    es_client="localhost:9200",
+...    es_index_pattern="flights",
+... )
+>>> df
+       AvgTicketPrice  Cancelled  ... dayOfWeek           timestamp
+0          841.265642      False  ...         0 2018-01-01 00:00:00
+1          882.982662      False  ...         0 2018-01-01 18:27:00
+2          190.636904      False  ...         0 2018-01-01 17:11:14
+3          181.694216       True  ...         0 2018-01-01 10:33:28
+4          730.041778      False  ...         0 2018-01-01 05:13:00
+...               ...        ...  ...       ...                 ...
+13054     1080.446279      False  ...         6 2018-02-11 20:42:25
+13055      646.612941      False  ...         6 2018-02-11 01:41:57
+13056      997.751876      False  ...         6 2018-02-11 04:09:27
+13057     1102.814465      False  ...         6 2018-02-11 08:28:21
+13058      858.144337      False  ...         6 2018-02-11 14:54:34
+
+[13059 rows x 27 columns]
+------------------------------------
+
+Eland can also be used for complex queries and aggregations:
+
+[source,python]
+------------------------------------
+>>> df[df.Carrier != "Kibana Airlines"].groupby("Carrier").mean(numeric_only=False)
+                  AvgTicketPrice  Cancelled                     timestamp
+Carrier                                                                  
+ES-Air                630.235816   0.129814 2018-01-21 20:45:00.200000000
+JetBeats              627.457373   0.134698 2018-01-21 14:43:18.112400635
+Logstash Airways      624.581974   0.125188 2018-01-21 16:14:50.711798340
+------------------------------------
+
+[discrete]
+=== License
+
+Licensed to Elasticsearch B.V. under one or more contributor
+license agreements. See the NOTICE file distributed with
+this work for additional information regarding copyright
+ownership. Elasticsearch B.V. licenses this file to you under
+the Apache License, Version 2.0 (the "License"); you may
+not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
```

Comment:
I would be in favor of removing the licensing info. We'll remove the `License` sections from the rest of the Client books, too.

Reply:
Got it!